recipes-drpai/gstreamer1.0-drpai/gstreamer1.0-drpai.bb

```diff
@@ -2,7 +2,7 @@ SUMMARY = "GStreamer DRP-AI plugin"
 SECTION = "multimedia"
 LICENSE = "MIT"
 SRC_URI = "git://github.com/MistySOM/gstreamer1.0-drpai.git;branch=master"
-SRCREV = "a7dee74b4b2c51d16a6dc6e5c1aa074cf08240bb"
+SRCREV = "${AUTOREV}"
```

Comment:
always pull from `HEAD`, is that safe? It appears safer to explicitly specify a tested and verified hash. The stuff in `master`  hasn't passed QA yet, necessarily, right?

Reply:
Yes but make a note for yourself to update this before it gets merged into `master`! Dangerous!

Reply:
You are right. This shouldn't be merged into `master`

Reply:
Yes but really `develop` should be the staging ground for what goes into `master` plus now that the drpai plugin has reached some stability, I'd imagine that update of the hash should not need to be that frequent anymore, right? I still don't feel like this should go into develop. If this is used in your dev branch, it appears okay. We have to work towards producing more stable releases that can be recreated at a later time (for support purposes). 
This change seems to work against that practice.